### PR TITLE
Refactor the /proto dir to be at the root level

### PR DESCRIPTION
**Pull request by eric-waldman**

### CODE REVIEW

File: browser/include/content_analysis/sdk/analysis_client.h [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #include <memory>
 #include <string>
 
-#include "content_analysis/sdk/analysis_agent.pb.h"
+#include "content_analysis/sdk/agent.pb.h"
```
Comment (rogerta) [inline browser/include/content_analysis/sdk/analysis_client.h:11]: Should this be analysis.pb.h ?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 .vscode/
+build/*
```
Comment (rogerta) [inline .gitignore:2]: Do we need the trailing `*`?  .gitignore is supposed to treat `build/` to mean ignore all files in this directory and below.

Do we need this line at all?  Probably best to build completely outside the source tree.

**Review by rogerta — COMMENTED** [review]
Thanks Eric. See a couple of comments below.

File: browser/include/content_analysis/sdk/analysis_client.h (hunk 1 above)
Comment (eric-waldman) [reply browser/include/content_analysis/sdk/analysis_client.h:11]: Yes it should!

File: .gitignore (hunk 2 above)
Comment (eric-waldman) [reply .gitignore]: The trailing `*` is not needed.

While there's certainly some advantages to having a read only source tree, it's also convenient to have the option to build in-tree under a `build/` directory.

**Review by rogerta — APPROVED** [review]
Thanks Eric!

**Pull request merged**